libraries/lib-theme/AColor.cpp

```diff
@@ -192,43 +270,39 @@ void AColor::Bevel(wxDC & dc, bool up, const wxRect & r)
    else
       AColor::Dark(&dc, false);
 
-   AColor::Line(dc, r.x, r.y, r.x + r.width, r.y);
-   AColor::Line(dc, r.x, r.y, r.x, r.y + r.height);
+   const auto left = r.x;
+   const auto right = r.x + r.width - 1;
+   const auto top = r.y;
+   const auto bottom = r.y + r.height - 1;
+
+   AColor::Line(dc, left, top, right, top);
+   AColor::Line(dc, left, top, left, bottom);
```

Comment:
Did you discover off by one mistakes in this function that you are also fixing?  A commit comment should say so.

Should you fix it?  Is it making small but noticeable changes in other drawing that wasnt' specified?

Reply:
You add no calls to Bevel.  Is it used in drawing functions that were changed?

Rather than change the more widely used function, adjust the rectangle that you pass into the function in the one relevant place.

Reply:
But I think there are no such uses that you need to adjust.